ball.py

```diff
@@ -1,5 +1,7 @@
-import pygame
 from random import randint
```

Comment:
we need randint for the ball generation of bounce

Reply:
just to fix the comment i tought its from random import randint that got removed.